docs/ensuring-repro/workflows/run-module-gha.md

```diff
@@ -3,7 +3,7 @@
 To maintain module functionality over time, we use [GitHub Actions](https://docs.github.com/en/actions) (GHAs) to periodically run each module ("module testing GHA") with the goal of testing that the module code runs to completion without errors.
 
 !!! info
-    For more information about how we run modules to generate official results and OpenScPCA releases, please see our documentation on the `OpenScPCA-nf` workflow. <!-- openscpca-nf STUB_LINK -->
+    For more information about how we run modules to generate official results and OpenScPCA releases, please see our [documentation on the `OpenScPCA-nf` workflow](../openscpca-nf/index.md).
```

Comment:
I don't love "official results" but I like the sentiment... Can we come up with another phrase that we can use more widely?

Reply:
Maybe "results for (intended for?) release", but that might a bit wordy?

Reply:
I didn't quite land at a better phrase, but did this rephrase instead. What do you think? https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/582/commits/1fc1b0cb4e84282f99dcb4bb0000f42259878ba1

Worth noting the phrase "official results" doesn't appear anywhere else, but also this change I made doesn't really give a phrase to use more generally either